frontend-react/src/hooks/UseOktaMemberships.ts

```diff
@@ -217,6 +222,27 @@ export const useOktaMemberships = (
     const token = authState?.accessToken;
     const organizations = authState?.accessToken?.claims?.organization;
 
+    const memberServices = useMemberServices(state, token);
+    // Update services when these values change
+    useEffect(() => {
```

Comment:
This is definitely out of the scope of this pull request, but I think it'd be a good idea to revisit our useEffect usages across the site.  There are a lot of cases in which we're using effects to trigger some other event based on a computed change, but with React v18's strict mode that's technically a discouraged behavior.

For this particular case, I think we could attach a [subscription](https://github.com/okta/okta-auth-js#authstatemanagersubscribehandler) to the OktaAuth instance and trigger whatever we need that way 🤷 

HOWEVER, like I said, this is definitely outside the scope of this pull request -- it's just something that I'd like the team to keep in mind when applicable.